packages/ember-metal/lib/error_handler.js

```diff
@@ -30,14 +30,6 @@ export function setOnerror(handler) {
 }
 
 let dispatchOverride;
-// dispatch error
-export function dispatchError(error) {
```

Comment:
This was previously only used by `RSVP.on('error', ...)` (after reverting https://github.com/emberjs/ember.js/pull/14898), and `RSVP.on('error', ...)` in this PR now does the logic of `dispatchError` directly (which makes it much clearer that this is RSVP error handling specific).